cmd/api/main.go

```diff
@@ -404,6 +380,7 @@ func createChainIDStack(
 	return chains.ChainStack{
 		Store:                 systemStore,
 		Registry:              registry,
+		EventProcessor:        ep,
```

Comment:
`chains.ChainStack` now also has the `EventProcessor`, this is needed to allow an external module to ask the `EventProcessor`, which is the last processed block. More on that later in the PR.